From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Tue, 22 Nov 2011 00:30:34 +0000
Subject: nice-path

---
 stdext/gzip.ml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/stdext/gzip.ml b/stdext/gzip.ml
index 805ae02..af070cc 100644
--- a/stdext/gzip.ml
+++ b/stdext/gzip.ml
@@ -34,7 +34,7 @@ type input_type =
 let lower_priority cmd args =
   let ionice="/usr/bin/ionice" in
   let ionice_args=["-c";"3"] in (*io idle*)
-  let nice="/bin/nice" in
+  let nice="/usr/bin/nice" in
   let nice_args=["-n";"19"] in (*lowest priority*)
   let extra_args=nice_args@[ionice]@ionice_args in
   let new_cmd=nice in
-- 
